CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ Documentation for hipSPARSE is available at
 
 * Added `azurelinux` OS name for correcting gfortran dependency
 
+### Optimized
+
+* Removed unused `GTest` dependency from benchmark
```

Comment:
Is benchmark a series of tests or a suite? Should it be capitalized or in code ticks?

Reply:
Will allow other reviewers to clarify this as I am not certain on appropriate style.

Reply:
The benchmark here is an executable (specifically named hipsparse-bench) which allows performance timing of the hipsparse routines.